proposals/0216-widget-support.md

```diff
@@ -522,62 +587,157 @@ This functional group should be provided to the app developers as a conditional
+To reduce complexity on the head unit, window duplication can be supported by the SDL libraries. This could allow more flexibility to display duplication for existing windows. However, this increases the number of RPCs to be sent by the app and the user might see a delay in window updates (duplication not being synchronized).
 
 In order to reduce state machine complexity for widget HMI levels on SDL core side, the HMI can take control of HMI level transitions for widgets. However this might cause different behavior per HMI implementation which could be confusing to developers as the behavior is not consistent.
 
-Instead of only having `DisplayCapability` it is possible to separate display capability and screen capability in `SystemCapability`. This may allow some benefits avoiding repeating display data when screens have changed. Also, it may allow `OnSystemCapabilityUpdated` to specific screens, instead of repeating all screens data.
-
 It is possible to have a manual subscription to display capabilities. This is definitely a possible solution as the SDL managers will perform the capabilities subscription. However, as this subscription will be made for 100% of the apps and as subscribing takes more time sending RPCs it was considered that autosubscription improves performance in this case. Without the automatic subscription, this redesign would perform worse than the current design of returning the information in a response.
 
-Another alternative, similar to above, allows manual subscritpion but with automatic notifications after `RegisterAppInterfaceResponse` and `SetDisplayLayout`, regardless of the subscription. This would allow applications to subscribe, in order to get notified on HMI changes related to the screen caused by the system or the user. Still it provides automatic notifications if the screen related HMI change is caused by the app (e.g. by changing the layout). 
+Another alternative, similar to above, allows manual subscritpion but with automatic notifications after `RegisterAppInterfaceResponse` and `SetDisplayLayout`, regardless of the subscription. This would allow applications to subscribe, in order to get notified on HMI changes related to the window caused by the system or the user. Still it provides automatic notifications if the window related HMI change is caused by the app (e.g. by changing the layout). 
 
-To avoid refactoring `SetDisplayLayout` to `SetScreenLayout` it is possible to just deprecate the parameters of the response.
+Instead of deprecating `SetDisplayLayout` RPC and adding a new paramter in `Show` RPC, we can refactor the RPC to be called `SetWindowLayout`. This would also prevent overburdenning of `Show` RPC. 
```

Comment:
```suggestion
Instead of deprecating `SetDisplayLayout` RPC and adding a new parameter in `Show` RPC, we can refactor the RPC to be called `SetWindowLayout`. This would also prevent overburdening of `Show` RPC. 
```